modify README (#1)
EverlyWell/backend-challenge: README.md

* Results should show the path of introduction from Alan to the expert e.g. Alan wants to get introduced to someone who writes about 'Dog breeding'. Claudia's website has a heading tag "Dog breeding in Ukraine". Bart knows Alan and Claudia. An example search result would be Alan -> Bart -> Claudia ("Dog breeding in Ukraine").

We encourage the use of any libraries for everything except the search functionality, in which we want to see your simple algorithm approach.

### Add-ons:

* Sign up/log in functionality
* A UI that expands upon the basic requirements to have a user-friendly look and feel
* Anything else you, as a user, would enjoy seeing in an interface like this

### Things we're looking for:

* Navigable code
* Efficient algorithms
* Good separation of concerns
* Error handling
* Usage of gems/libraries

### Things we like:

* Well commented & well organized code
* Quality over quantity (the code you write should be good) 
* Small, meaningful, commits
* Tests!
* Respect for the time limit - if you are in the midst of some work that you would like to finish, but have hit the 4 hour time limit, please split additional work into a separate branch, to be evaluated separately

> 

Reviewer: _Also cute animals_

Author: Hahaha, show us cute animals and we'll hire you.

### Submission

> * __Fork__ this repository to your own git
> * Create a new branch to push your commits as you work
> * When complete, create a PR to the master branch (of your personal repo) so we can see the code that you added!
> * Somehow share your repository with us

Reviewer: If we don't have scaffolding in this project then I don't think all these instructions are necessary? We could leave it as:
```suggestion
* __Fork__ this repository to your own git
* Remember to make meaningful commits as you work
* Somehow share your repository with us
```
